projects/components/src/refresh-button/refresh-button.service.ts

```diff
@@ -0,0 +1,12 @@
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs';
+
+@Injectable({ providedIn: 'root' })
```

Comment:
Not sure this makes sense as a global service - more so as a button output, since multiple things might want refreshing independently.

Reply:
Right.

Reply:
Removed the service.